packages/venia-ui/lib/components/CartPage/PriceAdjustments/giftCardSection.ee.js

```diff
@@ -15,7 +16,9 @@ const GiftCardSection = props => {
                 defaultMessage: 'Apply Gift Card'
             })}
         >
-            <GiftCards setIsCartUpdating={setIsCartUpdating} />
+            <Suspense fallback={null}>
```

Comment:
Why the lack of fallbacks in all these added usages? Happy path fast network would mean you'd likely never see it, but perhaps this is still reachable? Tiny example of toggling Suspense on/off with React DevTools:

![](https://gyazo.com/8e72aa45afe1083d18c64f1fe6ce655e.gif)

Reply:
I can add loading spinners. I just didn't think it was necessary since all the sections are behind user interaction. You're right though, we should probably include _something_ to indicate loading is happening.

Reply:
Intended to be a conversation starter, don't implement anything yet. I honestly haven't been able to catch an unloaded suspense component, even on Slow 3G, so if the use case isn't realistic, the `null`s are fine.

Reply:
It's easy enough to add them to these price adjustment accordions. Everything else though, I will not touch, including modals, or dropdowns, or the left nav.